tests/zfs-tests/tests/functional/cli_root/zfs_clone/zfs_clone_010_pos.ksh

```diff
@@ -217,16 +217,21 @@ log_note "verify clone list truncated correctly"
 typeset -i j=200
 i=1
 fs=$TESTPOOL/$TESTFS1
+if is_linux; then
+	ZFS_MAXPROPLEN=4096
+else
+	ZFS_MAXPROPLEN=1024
+fi
 log_must $ZFS create $fs
 log_must $ZFS snapshot $fs@snap
-while((i < 7)); do
+while((i < $(( $ZFS_MAXPROPLEN/200+1 )))); do
```

Comment:
$(( $ZFS_MAXPROPLEN/200+1 )) evaluates to 6 when ZFS_MAXPROPLEN=1024. The old value was 7. Is this change intentional, and if so, why? (I don't know what this code does, so I'm not saying it is wrong. I am just asking as part of reviewing this.)